src/views/Course.vue

```diff
@@ -0,0 +1,168 @@
+                        <span>
+                          {{
+                            item.deadline
+                              ? item.deadline
+                              : item.on_going
+                              ? 'On-going'
+                              : 'TBD'
+                          }}
+                        </span>
+                      </v-fade-transition>
+                    </v-col>
+                    <v-col cols="1">
+                      <v-fade-transition leave-absolute>
+                        <span>
+                          <v-btn icon>
+                            <v-icon class="mr-1">
+                              mdi-square-edit-outline
+                            </v-icon>
+                          </v-btn>
+                          <v-btn icon>
+                            <v-icon class="mr-1"> mdi-delete </v-icon>
+                          </v-btn>
+                        </span>
+                      </v-fade-transition>
+                    </v-col>
+                  </v-row>
+                </template>
+              </v-expansion-panel-header>
+              <v-expansion-panel-content>
+                <!-- TODO: Subtasks -->
+              </v-expansion-panel-content>
+            </v-expansion-panel>
+          </v-expansion-panels>
+        </div>
+      </v-sheet>
+    </v-col>
+  </v-row>
+</template>
+
+<script>
+import axios from 'axios';
+
+export default {
+  created() {
+    window.addEventListener('resize', this.handleResize);
+  },
+  data() {
+    return {
+      height: window.innerHeight,
+      file: null,
+      assessments: []
+    };
+  },
+  methods: {
+    handleResize() {
+      this.height = window.innerHeight;
+    },
+    onPickFile() {
+      this.$refs.fileInput.click();
+    },
+    onFilePicked(event) {
+      const { files } = event.target;
+      const filename = files[0].name;
+      const fileReader = new FileReader();
+      fileReader.addEventListener('load', () => {
+        this.fileUrl = fileReader.result;
+      });
+      fileReader.readAsDataURL(files[0]);
+      this.file = files[0];
+      if (window.confirm(`Do you want to use this syllabus: ${filename}?`)) {
+        const formData = new FormData();
+        formData.append('syllabus', this.file);
+        axios
+          .post(
+            `${process.env.VUE_APP_API_BASE_URL}/manager/parser`,
+            formData,
+            {
+              headers: {
+                'Content-Type': 'multipart/form-data',
+              },
+            },
+          )
+          .then((res) => {
+            for (const data of res.data) {
+              this.assessments.push({...data, grade: null})
+            }
+          })
+          .catch((err) => {
+            this.file = null;
+            console.error(err.message);
+          });
+      } else {
+        this.file = null;
+      }
+    },
+  },
+  computed: {
+    managerHeight() {
+      return this.height - 104;
```

Comment:
What does 104 represent?

Reply:
i think i copy pasted it from other views files, so not exactly sure

Reply:
Ah, in that case, it should be fine as it is but can you leave a todo comment to add docs to it